Projects/ChangeTier/Sources/Models/ChangeTierIntentModel.swift

```diff
@@ -40,83 +44,80 @@ public struct Tier: Codable, Equatable, Hashable, Identifiable {
     public var id: String
     let name: String
     let level: Int
-    public let deductibles: [Deductible]
-    let displayItems: [TierDisplayItem]
+    public let quotes: [Quote]
     let exposureName: String?
-    let productVariant: ProductVariant?
-    let FAQs: [FAQ]?
 
     public init(
         id: String,
         name: String,
         level: Int,
-        deductibles: [Deductible],
-        displayItems: [TierDisplayItem],
-        exposureName: String?,
-        productVariant: ProductVariant?,
-        FAQs: [FAQ]?
+        quotes: [Quote],
+        exposureName: String?
     ) {
         self.id = id
         self.name = name
         self.level = level
-        self.deductibles = deductibles
-        self.displayItems = displayItems
+        self.quotes = quotes
         self.exposureName = exposureName
-        self.productVariant = productVariant
-        self.FAQs = FAQs
-    }
-
-    public struct TierDisplayItem: Codable, Equatable, Hashable {
-        public var id = UUID()
-
-        public init(
-            title: String,
-            subTitle: String?,
-            value: String
-        ) {
-            self.title = title
-            self.subTitle = subTitle
-            self.value = value
-        }
-
-        let title: String
-        let subTitle: String?
-        let value: String
     }
 
     func getPremium() -> MonetaryAmount? {
-        if deductibles.count == 1 {
-            return deductibles.first?.premium
+        if quotes.count == 1 {
+            return quotes.first?.premium
         }
         return nil
     }
 }
 
-public struct Deductible: Codable, Hashable, Identifiable {
+public struct Quote: Codable, Hashable, Identifiable {
     public var id: String
-
-    let deductibleAmount: MonetaryAmount?
-    let deductiblePercentage: Int?
+    let quoteAmount: MonetaryAmount?
```

Comment:
This should still remain deductibleAmoount and deductiblePercentage because it is deductible related to the quote